### DISCUSSION TITLE
升级0.3.0版本websocket报错

### DISCUSSION THREAD

**Discussion by wyqjs** [web-infra-dev/rsbuild#1232] [Q&A]
![企业微信截图_49e78e10-6935-45f8-aa8c-62281837dc4b](https://github.com/web-infra-dev/rsbuild/assets/95618257/1a85f384-6c0d-4548-a1a9-518b9c6bfa00)

**Comment by chenjiahan** [comment]
Can you provide an reproduce repository for this issue?
